File: sistema.py
def cargar_persona():
    nombre = input("Ingrese nombre: ")
    apellido = input("Ingrese apellido: ")
    dni = input("Ingrese DNI: ")
    
    telefonos = []
    while True:
        telefono = input("Ingrese un teléfono (o presione Enter para finalizar): ")
        if telefono:
            telefonos.append(telefono)
        else:
            break

    return [nombre, apellido, dni, telefonos]

def mostrar_personas(personas):
    print("\nLista de Personas Registradas:")
    for persona in personas:
        print(persona)

def main():
    personas = []
    while True:
        personas.append(cargar_persona())
        continuar = input("¿Desea ingresar otra persona? (s/n): ").strip().lower()
        if continuar != 's':
            break

    mostrar_personas(personas)  # 🔹 Aquí pasamos la lista como argumento

if __name__ == "__main__":
    main()

File: sistema.js
const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

let personas = [];

function cargarPersona() {
    rl.question('Ingrese nombre: ', (nombre) => {
        rl.question('Ingrese apellido: ', (apellido) => {
            rl.question('Ingrese DNI: ', (dni) => {
                let telefonos = [];
                
                function agregarTelefono() {
                    rl.question('Ingrese un teléfono (o presione Enter para finalizar): ', (telefono) => {
                        if (telefono !== '') {
                            telefonos.push(telefono);
                            agregarTelefono();
                        } else {
                            personas.push([nombre, apellido, dni, telefonos]);
                            rl.question('¿Desea ingresar otra persona? (s/n): ', (respuesta) => {
                                if (respuesta.toLowerCase() === 's') {
                                    cargarPersona();
                                } else {
                                    mostrarPersonas();
                                    rl.close();
                                }
                            });
                        }
                    });
                }

                agregarTelefono();
            });
        });
    });
}

function mostrarPersonas() {
    console.log('\nLista de Personas Registradas:');
    personas.forEach(persona => {
        console.log(persona);
    });
}

cargarPersona();

